zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -272,15 +276,17 @@ impl<F: Field> SubCircuitConfig<F> for BytecodeCircuitConfig<F> {
                 meta.query_advice(length, Rotation::cur()),
             );
 
-            let empty_hash = rlc::expr(
-                &EMPTY_CODE_HASH_LE.map(|v| Expression::Constant(F::from(v as u64))),
-                challenges.evm_word(),
-            );
+            let empty_hash_word: Word<Expression<F>> =
+                Word32::new(EMPTY_CODE_HASH_LE.map(|v| Expression::Constant(F::from(v as u64))))
+                    .to_word();
 
-            cb.require_equal(
+            cb.require_equal_word(
```

Comment:
All of `Word<Expression<F>>` query can apply below less verbose version, e.g.
```diff
- Word::new([
-                    meta.query_advice(bytecode_table.code_hash.lo(), - Rotation::next()),
-                    meta.query_advice(bytecode_table.code_hash.hi(), - Rotation::next()),
-                ])
+ bytecode_table
+          .code_hash
+          .map(|column| meta.query_advice(column, Rotation::cur())),
```

Reply:
My opinion is that this is less radable.

Could we perhaps add something to the Word interface?

```
impl Word {
  fn query_advice(meta, Word<Column>) -> WordExpr
}
```

Reply:
Sorry, there should be a rotation parameter as well.

Reply:
perhaps

```

impl Word<Column<Advice>> {
   fn query_advice<F: Field>(&self, meta, rotation) -> Word<Expression<F>>
}
```

Reply:
> perhaps
> 
> ```
> 
> impl Word<Column<Advice>> {
>    fn query_advice<F: Field>(&self, meta, rotation) -> Word<Expression<F>>
> }
> ```

👍 nice proposal to even hide `limbs` concept by dedicate `query_advice`  function

Reply:
please, check now